### ISSUE TITLE
Broken link in README.md

### ISSUE THREAD

**Issue by tcmoore3** [glotzerlab/software#36]
## Description

The very last link under the "Contributing" section (to CONTRIBUTING.md) links to a missing file. Perhaps it should link to ARCHITECTURE.md?

**Comment by joaander** [comment]
No, not `ARCHITECTURE.md`. This is a copy and paste bug from HOOMD or some other source. Those projects have clear contributor guidelines: https://github.com/glotzerlab/hoomd-blue/blob/master/CONTRIBUTING.md

We either need to write such a document for `glotzerlab/software` or remove the link.

**Comment by tcmoore3** [comment]
I see. In that case, it should probably be removed unless there is a strong need for contributor guidelines, which I don't see (at least for now).

**Comment by joaander** [comment]
I didn't see the PR you started. Feel free to link to `ARCHITECTURE.md` from there. I'd reword it so that it sounds more like a link to developer documentation than a contributor guide.
